portfolio/src/main/webapp/style.css

```diff
@@ -36,6 +37,53 @@ h1 {
 
 * {
   box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.titlecontainer{
```

Comment:
nit: I've always seen CSS with a space between the declaration and the '{'.

Reply:
got it, fixed